tests/api_test.py

```diff
@@ -507,20 +507,30 @@ def vjpfun(x, y):
       vjp = lambda g: (g + x + y, g * x * 9.)
       return out, vjp
 
-    defvjp_all(foo_p, vjpfun)
+    ad.defvjp_all(foo_p, vjpfun)
     val_ans, grad_ans = api.value_and_grad(foo)(3., 4.)
     self.assertAllClose(val_ans, 3.**2 + 4.**3, check_dtypes=False)
     self.assertAllClose(grad_ans, 1. + 3. + 4., check_dtypes=False)
 
     ans = api.grad(foo, (0, 1))(3., 4.)
     self.assertAllClose(ans, (1. + 3. + 4., 1. * 3. * 9.), check_dtypes=False)
 
+  def test_defvjp_all(self):
+    @api.custom_transforms
+    def foo(x):
```

Comment:
Maybe add a test for a function that takes multiple and/or pytree arguments? (Sorry if I missed it below)

Reply:
(This can be a follow-up change, but I think is worth doing, at the very least to make sure we have a runnable example of multiple args.)

Reply:
Will do in a follow-up; left a todo.